### PR TITLE
Add smoke-build CI workflow

### PR DESCRIPTION
**Pull request by IanVS**
This adds a workflow into PRs to build the iOS and Android apps, just to verify that they aren't completely broken / unable to build.  They're debug builds, so it won't completely mirror the release build, but that allows us to avoid getting signing keys in the workflow.

### CODE REVIEW

**Comment by jasikpark** [comment]
bravo 🎉

**Review by jasikpark — APPROVED** [review]
lgtm, without knowledge of how this all works, but it seems reasonable, plus green CI looks good 👍

File: ios/fastlane/Fastfile [hunk 1]
```diff
@@ -18,6 +18,50 @@ default_platform(:ios)
 platform :ios do
   desc "Push a new beta build to TestFlight"
 
+  lane :checkBuild do
```
Comment (nbrownus) [inline ios/fastlane/Fastfile:21]: Do we need the signing credentials available? If yes, if we switch this to debug can we avoid it?

File: nebula/Makefile [hunk 2]
```diff
@@ -9,7 +9,7 @@ clean:
 
 mobileNebula.aar: *.go go.sum
 	go get -d golang.org/x/mobile/cmd/gomobile
-	gomobile bind -trimpath -v --target=android -androidapi 26
+	gomobile bind -trimpath -v --target=android -androidapi=26
```
Comment (IanVS) [inline nebula/Makefile:12]: this is dumb but might as well keep it consistent with `--target`

**Comment by IanVS** [comment]
I wish we had this prior to the last PR being merged, which upgraded nebula but caused the build to break.  Working on fixing it now, but would love a review on this PR in the meantime.

**Review by nbrownus — APPROVED** [review]

**Pull request merged**